pipeline-modules/infra-service/aws/git-access.yaml

```
@@ -46,5 +46,5 @@ template: |
         - git config --file {{ git_config_path }} credential.helper 'store --file {{ git_creds_path }}'
         - git config --file {{ git_config_path }} credential.useHttpPath true
         - git config --file {{ git_config_path }} credential.interactive never
-        - git config --global url."https://{{ git_login }}:$(printf %s $GIT_PASSWORD | jq -sRr @uri)@{{ git_remote_path }}".insteadOf "https://{{ git_remote_path }}"
+        - git config --global url."sed -e "s^//^//{{ git_login }}:$(printf %s $GIT_PASSWORD | jq -sRr @uri)@^" <<< {{ git_remote_path }}".insteadOf "{{ git_remote_path }}"
```

```suggestion
        - git config --global url."$(sed -e "s^//^//{{ git_login }}:$(printf %s $GIT_PASSWORD | jq -sRr @uri)@^" <<< {{ git_remote_path }})".insteadOf "{{ git_remote_path }}"
```

sed becomes part of string here where as we need it to run. so to do that, we just wrap it with `$(...)`

---

done